docs/getting-started/create-comit-app.md

```diff
@@ -4,6 +4,286 @@ title: Getting Started
+│       │   └── index.ts
+│       ├── tsconfig.json
+│       ├── tslint.json
+│       └── yarn.lock
+├── examples
+│   ├── README.md
+│   └── btc_eth
+│       ├── README.md
+│       ├── package.json
+│       ├── src
+│       │   ├── lib.ts
+│       │   ├── maker.ts
+│       │   └── taker.ts
+│       ├── tsconfig.json
+│       ├── tslint.json
+│       └── yarn.lock
+├── package.json
+└── src
+    └── index.js
+
+9 directories, 24 files
+```
+
+- `.` is where you can start writing your app, see [write your app tuto](TODO) 
+- `./package.json` contains some useful dependencies for your app
+- `./examples` contains projects that are made to inspire developers such as yourself to build their app
+- `./demos` contains projects which are catered for workshops demonstration. They do not have a conventional structure but have verbose output.
+ 
+ Let's have a look at [comit-scripts](#comit-scripts) before [running an example](TODO) together.
+
+## comit-scripts
+
+The `comit-scripts` package helps you run a development environment to build a COMIT App.
+
+Let's install it from the template folder to play with it.
+
+```
+▶ cd my-app # Not needed if you are already in the `my-app` directory
+▶ yarn install
+yarn install v1.21.1
+info No lockfile found.
+[1/4] 🔍  Resolving packages...
+[2/4] 🚚  Fetching packages...
+[3/4] 🔗  Linking dependencies...
+[4/4] 🔨  Building fresh packages...
+success Saved lockfile.
+✨  Done in 92.81s.
+```
+
+Let's check out the provided `package.json` for any useful scripts:
+
+```
+▶ cat package.json
+{
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true,
+  "main": "index.js",
+  "scripts": {
+    "start": "node ./src/index.js",
+    "start-env": "comit-scripts start-env",
+    "force-clean-env": "comit-scripts force-clean-env"
+  },
+  "dependencies": {
+    "comit-sdk": "^0.10.0"
+  },
+  "devDependencies": {
+    "comit-scripts": "^0.8.3"
+  }
+}
+```
+
+As you can see, two scripts are using `comit-scripts`, let's run `start-env`:
+
+```
+▶ yarn run start-env
+yarn run v1.21.1
+$ comit-scripts start-env
+Creating Docker network (create-comit-app)...✓
+Starting Ethereum node...✓
+Starting Bitcoin node...✓
+Starting two cnds...✓
+Writing configuration in env file...✓
+🎉 Environment is ready, time to create a COMIT app!
+✓
+```
+
+So, what does it do?
+
+`Creating Docker network (create-comit-app)...✓`: creates a docker network to allow several docker containers to communicate with each others. The network is named `create-comit-app`.
+
+
+`Starting Ethereum node...✓`:
+- Starts a parity node in a docker container,
+- Deploys an ERC20 token contract
+- Creates two accounts,
+- Funds each account with 1000 Ethers and 100 ERC20 tokens.
+
+`Starting Bitcoin node...✓`"
+- Starts a bitcoind node in a docker container,
+- Create two HD wallet,
+- Funds each wallet with 10 Bitcoin,
+- Run `generate` every second to mine a new block.
+
+`Starting two cnds...✓`: Starts 2 cnd (COMIT Network Daemon) in docker containers.
```

Comment:
Like stated at the beginning of the section above, I think this really needs a bit more explanation why this is done. (Why are 2 cnd nodes needed...) 
This is the very first thing that people see. In my opinion we cannot assume that they already know what is going on. We can link to other documentation if it will be described there.

Reply:
I'd argue that the very first document they see is the overall overview.

Reply:
Done.